src/scanner.py

```diff
@@ -171,18 +173,25 @@ def run(self) -> NoReturn:
         Main Loop of the Scanner
         """
         log.info("Scanner started ...")
-        while True:
+        if (self.config.schedule_cron != '* * * * *'):
             try:
-                self._job()
-                if self.tgtg_client.captcha_error_count > 10:
-                    log.warning("Too many 403 Errors. Sleeping for 1 hour.")
-                    sleep(60 * 60)
-                    log.info("Continuing scanning.")
-                    self.tgtg_client.captcha_error_count = 0
+                log.info("Schedule cron expression: " + get_description(self.config.schedule_cron))
             except Exception:
-                log.error("Job Error! - %s", sys.exc_info())
-            finally:
-                sleep(self.config.sleep_time * (0.9 + 0.2 * random()))
+                log.warning("Schedule cron expression parsing error - %s", sys.exc_info())
+                log.info("Schedule cron expression is ignored")
+                self.config.schedule_cron = '* * * * *'
+        while True:
+            if (pycron.is_now(self.config.schedule_cron)):
```

Comment:
Can this line raise an error?

Reply:
I'll have to look into the code of `pycron` to know for sure. The doc for the lib is rather thin. I'll get back to you on that.

Reply:
Looked into library `pycron` and `cron_descriptor` source code, Resulted in some changes arround multible places. [5cfeb2a](https://github.com/Der-Henning/tgtg/pull/142/commits/5cfeb2aa9b8d502bf4d599cb24b17cf8ffdd6755).
Testing this branch now via `docker pull zinen2/tgtg:edge-alpine``.